app/_src/production/cp-deployment/multi-zone.md

```diff
@@ -192,85 +192,105 @@ You need the following values to pass to each zone control plane setup:
 {% tabs zone-control-planes useUrlFragment=false %}
 {% tab zone-control-planes Kubernetes %}
 
-**Without zone egress**:
-
 1.  On each zone control plane, run:
 
+    {% if_version gte:2.3.x %}
     ```sh
     kumactl install control-plane \
     --mode=zone \
     --zone=<zone name> \
     --ingress-enabled \
-    --kds-global-address grpcs://<global-kds-address>:5685 | kubectl apply -f -
+    --kds-global-address grpcs://<global-kds-address>:5685 \
+    --set controlPlane.tls.kdsZoneClient.skipVerify=true | kubectl apply -f -
     ```
-
-    where `zone` is the same value for all zone control planes in the same zone.
-
-**With zone egress**:
-
-1.  On each zone control plane, run:
-
+    {% endif_version %}
+    {% if_version lte:2.2.x %}
     ```sh
     kumactl install control-plane \
     --mode=zone \
-    --zone=<zone-name> \
+    --zone=<zone name> \
     --ingress-enabled \
-    --egress-enabled \
     --kds-global-address grpcs://<global-kds-address>:5685 | kubectl apply -f -
     ```
+    {% endif_version %}
 
     where `zone` is the same value for all zone control planes in the same zone.
 
+    Add `--egress-enabled` to list of arguments if you want to deploy optional [Zone Egress](/docs/{{ page.version }}/production/cp-deployment/zoneegress/).
+
+    {% if_version gte:2.3.x %}
+    `--set controlPlane.tls.kdsZoneClient.skipVerify=true` is required because the default global control plane's certificate is self-signed.
+    It is recommended to use a certificate signed by a trusted CA in production. See [Secure access across services](/docs/{{ page.version }}/production/secure-deployment/certificates/) page for more information.
```

Comment:
I guess maybe this to not use passive voice? :

```suggestion
    We recommend to use a certificate signed by a trusted CA in production. See [Secure access across services](/docs/{{ page.version }}/production/secure-deployment/certificates/) page for more information.
```

Reply:
It does not like We as much as passive voice